flex/http.py

```diff
@@ -130,11 +131,16 @@ def _normalize_tornado_request(request):
     if not _tornado_available:
         raise TypeError("Tornado is not installed")
 
-    if not isinstance(request, tornado.httpclient.HTTPRequest):
+    if isinstance(request, tornado.httpclient.HTTPRequest):
+        url_attr = 'url'
+    elif isinstance(request, tornado.httpserver.HTTPRequest):
+        # This is the only difference in their api that we care about.
+        url_attr = 'uri'
```

Comment:
how about just pulling the URL off of the `request` and removing the `getattr` from the return statement.